### ISSUE TITLE
Player Options > Infinite Stamina & Lung Capacity

### ISSUE THREAD

**Issue by TayMcKenzieNZ** [itsjustcurtis/MenyooSP#64]
Unless I am blind, I could have sworn in previous versions of Menyoo, there was an option for Infinite Stamina and lung capacity *(or whatever that is called)*. I only see Invincibility, Invisibilty, No Ragdoll, etc etc. Maybe there is something there but commented out?

**Comment by mistahwolf** [comment]
Yeah, I don't remember ever seeing an 'infinite stamina and lung capacity' toggle in menyoo since using it from back in '17. You do get that when you select 'invincibility' in Menyoo. You have more options for that with 'Simple Trainer' and 'Enhanced Native Trainer.' Both have the 'unlimited stamina' option and 'Simple' has a 'Special Godmode' which allows you to take damage from drowning, bullets, catching fire etc. w/out dying outright. Explosions however.... A similar mode in Menyoo would be nice.
